test/DeployBase.t.sol

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.23;
+
+import {Test} from "forge-std/Test.sol";
+import {IEarningPowerCalculator} from "../src/interfaces/IEarningPowerCalculator.sol";
+import {MintRewardNotifier} from "../src/notifiers/MintRewardNotifier.sol";
+import {TransferRewardNotifier} from "../src/notifiers/TransferRewardNotifier.sol";
+import {Staker} from "../src/Staker.sol";
+import {DeployBaseFake} from "./fakes/DeployBaseFake.sol";
+import {DeployMultipleRewardNotifiersFake} from "./fakes/DeployMultipleRewardNotifiersFake.sol";
+import {ERC20Fake} from "./fakes/ERC20Fake.sol";
+import {ERC20VotesMock} from "./mocks/MockERC20Votes.sol";
+
+contract DeployBaseTest is Test {
+  ERC20Fake rewardToken;
+  ERC20VotesMock govToken;
+  DeployBaseFake deployScript;
+
+  function setUp() public {
+    rewardToken = new ERC20Fake();
+    vm.label(address(rewardToken), "Reward Token");
+
+    govToken = new ERC20VotesMock();
+    vm.label(address(govToken), "Governance Token");
+
+    deployScript = new DeployBaseFake(rewardToken, govToken);
```

Comment:
Since we only use this in one test, is it better to just deploy it as a local variable in that test?

Reply:
Good catch!@\

Reply:
https://github.com/withtally/staker/pull/126/commits/8e6c26d297d2681135bbdde81054842189fefb3d